focil.md

```diff
@@ -0,0 +1,16 @@
+# FOCIL Metrics
+
+The following metrics are proposed to be added to clients for FOCIL monitoring. This list is open for discussion. Each client has the opportunity to contribute to it by suggesting additions or disputing existing metrics.
+
+| Name | Metric type | Usage | Sample collection event |
```

Comment:
How about also adding the number of equivocated ILs and the number of valid ILs per slot (histogram)?

Reply:
IMHO, it would be nice if we could have multiple metrics regarding the number of ILs that are submitted, valid, invalid (wrong signature, wrong IL committee root, late submission, etc) and equivocating. There are clear relationships among these metrics and it would give us good overview of the IL health.